src/test/store/profile-view.test.js

```diff
@@ -116,83 +123,613 @@ describe('call node paths on implementation filter change', function() {
   });
 });
 
-describe('expand all call node descendants', function() {
-  const {
-    profile,
-    funcNamesPerThread: [funcNames],
-  } = getProfileFromTextSamples(`
-    A A A
-    B B E
-    C D
-  `);
-  const threadIndex = 0;
-  const A = funcNames.indexOf('A');
-  const B = funcNames.indexOf('B');
-  const C = funcNames.indexOf('C');
-  const D = funcNames.indexOf('D');
-  const E = funcNames.indexOf('E');
+/**
+ * The following tests naively runs through a dispatch and selector to provide coverage
```

Comment:
The word "naively" is strange here.
also, nit: `runs` => `run`.

Reply:
Ok, I tried to explain better what my intent was.
```
/**
 * The following tests run through a dispatch and selector to provide coverage
 * over the Redux store to ensure that it behaves correctly. The intent is to do
 * the bare minimum to assert the relationship between the actions and reducers.
 */
```